apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -237,6 +238,13 @@ function ListBoxInline({ options, layout }) {
     containerPadding = layoutOptions.layoutOrder === 'row' ? '2px 4px' : '2px 6px 2px 4px';
   }
 
+  const TitleComponent = constraints?.active ? (
```

Comment:
there need also be a check of permission if the user has can do setProperties on the object